### PR TITLE
Greek Translation

### PR DESCRIPTION
**Pull request by pankgeorg**
# Notes:
I've kept the language as simple as possible, tried to use terms as used by [Prof Tsiodras](https://www.nytimes.com/2020/04/05/world/europe/scientists-coronavirus-heroes.html), added an extra reference to him in the valve mask argument and tried to reference greek wikipedia in the 1 case it was possible.

I looked at Twitter photos to assume the gender of people when I had to, but please let me know to update if. 
```
"ο" -> he
"η" -> she
```
Added the 'Tinos' font, because Raleway doesn't support greek.

### CODE REVIEW

**Comment by pankgeorg** [comment]
@ismarou I've added a PR, feel free to comment and/or send suggestions!

File: index-el.html
Comment (ismarou) [inline index-el.html:58]: I would change 'Τωρα γνωρίζουμε' with 'Γνωρίζουμε πλέον'

File: index-el.html
Comment (ismarou) [inline index-el.html:62]: 'για να σταματήσουμε μια επιδημια' ---> 'για να σταμαγτήσουμε μια πανδημία'

File: index-el.html
Comment (ismarou) [inline index-el.html:573]: εκφράζοτνας ---> εκφράζοντας in the paragram 'Το Ανθρώπινο Κράτος'

Comment (ismarou) [reply index-el.html:573]: paragraph*

File: index-el.html
Comment (ismarou) [inline index-el.html:649]: In general, I would change every 'επιδημία' with the word 'πανδημία', as this is one of the strongest arguments of people who refuse to wear them

File: index-el.html
Comment (pankgeorg) [reply index-el.html:58]: agreed

File: index-el.html
Comment (pankgeorg) [reply index-el.html:573]: fixed, thanks!

File: index-el.html
Comment (pankgeorg) [reply index-el.html:62]: @ismarou This change is best to be proposed to the original english text, which uses only the term "epidemic". If you feel that's important I'd think it's best to open an issue on the English text. We'll translate the original as is then.

File: index-el.html
Comment (pankgeorg) [reply index-el.html:649]: see comment above.

File: index-el.html
Comment (ismarou) [reply index-el.html:62]: I get what you say. Opening the issue in the English text would cause the extra problem/need of adapting all other translations, previous to ours, so I don't know how viable it really is. Since you have done all the work, it's your call to choose if you wish to be exact in terms of translation or semantics.

**Comment by aatishb** [comment]
Thanks @pankgeorg for working in this & @ismarou for reviewing! If it’s fine with you, I’ll go ahead and merge the latest changes into this branch, and will let you know if there’s anything left to do.

File: index-el.html
Comment (aatishb) [inline index-el.html:618]: Could you also translate this? (This is for the hover text in the second graph)

**Comment by aatishb** [comment]
Hi @pankgeorg, Thanks for your efforts on this. I merged the recent updates with this branch to bring it closer to merging. Here are the remaining steps:

- [ ] translate '% Infected'
- [ ] (optional) you're encouraged to add a line of credit & link to yourself for doing the translation, after the first sentence in the Credits/References 

One last thing I wanted to check is if Greek doesn't have gender neutral pronouns or articles for people, it would be good to use a roughly equal mix of male / female pronouns/articles, or even a greater fraction of female pronouns/articles, for places where the gender is unspecified (e.g. 'the contagious person'). I just want to make sure we don't use male pronouns/articles as the default when gender is unspecified. (It seems you may have already looked out for this?)

Thanks!

**Comment by aatishb** [comment]
Tagging https://github.com/aatishb/maskmath/issues/4

**Comment by pankgeorg** [comment]
@aatishb

- [x] Translated the `% infected`
- [x] Fixed some typos
- [x] Tried to gender-neutralize the text

In most of the text, I'm using `ο άνθρωπος` which is `the human being`, which unfortunately is male in Greek.
The status quo is to omit the `άνθρωπος` `[*]` and just refer to everyone as `ο`. I've added all instances of `άνθρωπος` where I'm using the male gender.

I'm also using `κάποιος` which means `someone`, which is intrinsicly male `(he/him)` and has the meaning of `some (hu)man` but the `άνθρωπος` is omitted (due to `[*]`) . In gender neutral texts this is expanded abbreviating the female version of `someone` to `κάποιος/α`, which i don't like because it is like 'default male 7 letters /i remembered female + 1`. I though that just `κάποιος` (with the omitted meaning of `άνθρωπος` was good enough but, on second though, I'm going with `κάποια ή κάποιος`, which is `someone (she/her) or someone (he/him)` which sounds the least weird and communicates the message.

I've pushed the respective branches, so if you're ok with this approach, you can proceed with the merge.

Thanks,
Panagiotis

**Comment by pankgeorg** [comment]
Also, on an offtopic note, I've also translated the [video](https://www.youtube.com/watch?v=Y47t9qLc9I4) but it doesn't show up (I think it needs some approval from minutephysics?) Do you know what process I have to follow/if/how can I contact Henry?

**Review by aatishb — APPROVED** [review]
Thanks @pankgeorg, I'll do one last test and proceed to merge.

**Comment by aatishb** [comment]
@pankgeorg re: video translation, I'll pass that on to Henry. (and thanks!) He's reachable via this page: https://www.minutephysics.com/contact

**Comment by aatishb** [comment]
@all-contributors add @pankgeorg for translation

**Comment by aatishb** [comment]
@all-contributors add @ismarou for review

**Comment by aatishb** [comment]
Thanks @pankgeorg @ismarou for working on this! Merged.

**Pull request merged**
